cmd/primateRecon/primateRecon.go

```diff
@@ -111,30 +83,47 @@ func hcaIsPresent(human, bonobo, chimp, gorilla, organutan dna.Base) bool {
 	return false
 }
 
+func hgaIsPresent(human, bonobo, chimp, gorilla, orangutan dna.Base) bool {
+	if baseIsPresent(gorilla) && (baseIsPresent(human) || baseIsPresent(chimp) || baseIsPresent(bonobo)) {
```

Comment:
The if conditions in hgaIsPresent are different than the way the if conditions are formulated in hcaIsPresent, but if I am understanding correctly this corresponds to the position of gorilla in the human-chimp-gorilla-bonobo-orangutan phylogeny tree and so is normal?

Reply:
yes that's the intuition